renderer/lib/cast.js

```diff
@@ -344,6 +345,17 @@ function pause () {
   }
 }
 
+function setRate (rate) {
```

Comment:
I'd make this function return `true` or `false` depending on whether this cast target supports setting the playback rate.